tokio/tests/task_hooks.rs

```diff
@@ -0,0 +1,76 @@
+#![allow(unknown_lints, unexpected_cfgs)]
+#![warn(rust_2018_idioms)]
+#![cfg(all(feature = "full", tokio_unstable, target_has_atomic = "64"))]
```

Comment:
Is there any code that will not work unless `target_has_atomic = "64"`?

Reply:
I originally used AtomicU64 for these tests, which was why it was added. I can remove this and see if it still passes CI.